lms/static/sass/course/_student-notes.scss

```diff
@@ -45,6 +45,12 @@ $divider-visual-tertiary: ($baseline/20) solid $gray-l4;
       @extend .content; // needed extend carried over from course handouts UI, but should be cleaned up
       width: 100%;
     }
+
+    .sr-is-focusable,
```

Comment:
@marcotuts @clrux added CSS like this (to hide the focus indicator, which we discussed after standup) to _courseware.scss. Because of scoping, the rules do not apply to the Student Notes page. Do you have a better place for me to put these styles? Should we put it at an application-wide level and remove the styles in _courseware.scss?